### PR TITLE
Doctrine cookbook

### PR DESCRIPTION
**Pull request by 1ma**
A new cookbook entry describing how to integrate Doctrine into Slim.

### CODE REVIEW

File: docs/cookbook/database-doctrine.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+<figure>
+{% highlight php %}
+<?php
+
+// bin/doctrine.php
+
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\Tools\Console\ConsoleRunner;
+
+/** @var \Slim\Container $cnt */
```
Comment (geggleto) [inline docs/cookbook/database-doctrine.md]: should this not be `$container`

**Review by geggleto — CHANGES_REQUESTED** [review]

File: docs/cookbook/database-doctrine.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        $container['settings']['doctrine']['conn'], $config
+    );
+};
+{% endhighlight %}
+<figcaption>Figure 3: Defining the EntityManager service.</figcaption>
+</figure>
+
+## Create the Doctrine console
+
+To be able to automatically run database migrations, validate class annotations and so on you need to create the Doctrine CLI application.
```
Comment (geggleto) [inline docs/cookbook/database-doctrine.md]: The console should be placed in the root of the project in the `cli-config.php` file (as per standard doctrine)

**Comment by geggleto** [comment]
I'll review this weekend

**Comment by akrabat** [comment]
Merged in https://github.com/slimphp/Slim-Website/commit/eb93493274d888447e5f3d18d4be41dc43699b6f

**Comment by akrabat** [comment]
Thanks @1ma.

**Pull request closed without merging**